src/lib/plugins/redstone.js

```diff
@@ -64,7 +68,7 @@ module.exports.server = function (serv, { version }) {
 
   const isDirectedRepeater = (block, dir, powered = false) => {
     if (block.type !== poweredRepeaterType &&
-        (block.type !== unpoweredRepeaterType || powered)) return false
+        (block.type !== unpoweredRepeaterType || block.type !== repeaterType || powered)) return false
```

Comment:
I think a cleaner way would be to redefine the whole function, something like:
```js
const isDirectedRepeater = !serv.supportFeature('theFlattening') ?
(block, dir, powered = false) => {
  // current code
} :
(block, dir, powered = false) => {
 // 1.13+ code
}
```
Same for all the type helper functions:
* powerLevel
* powerLevelDir
* isWireDirectedIn
* etc..

Reply:
Now that we cant write it in a single line, doesnt it make more sense to use a more common syntax? The code you wrote is complex/weird, isnt it?

Reply:
yeah I tried to be minimalist so you get the idea, in reality you would want to split in different variables

Reply:
done